app/gui/src/dashboard/components/styled/Tabs.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * @file
+ *
+ * This file is a modified version of the `Tabs` components from the
```

Comment:
is there a reason this is in `styled` and not `AriaComponents`?

Reply:
No reason, I think the ideal place would be `#/component/aria`. Thoughts?

Reply:
yup, makes sense